### PR TITLE
Changes pending

### PR DESCRIPTION
**Pull request by Gaspela**

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -77,13 +77,13 @@ def get(self, cls, id):
             return self.all(cls).get(cls + '.' + id)
 
     def count(self, cls=None):
-        """ Return number objs in strg depend match of class name """
+        """  """
```
Comment (sebaslherrera) [inline models/engine/db_storage.py]: Add comments for count method and description of variables

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -77,13 +77,13 @@ def get(self, cls, id):
...
         if cls:
             return len(self.all(cls))
         else:
             return len(self.all())
 
 
     def close(self):
-        """call remove() method on the private session attribute"""
+        """ """
```
Comment (sebaslherrera) [inline models/engine/db_storage.py]: Add comments of close method

File: models/engine/file_storage.py [hunk 3]
```diff
@@ -71,7 +71,7 @@ def get(self, cls, id):
             return self.all(cls).get(cls + '.' + id)
 
     def count(self, cls=None):
-        """ Return number objs in strg depend match of class name """
+        """  """
```
Comment (sebaslherrera) [inline models/engine/file_storage.py]: Add comments of count method

**Review by sebaslherrera — CHANGES_REQUESTED** [review]

File: models/engine/db_storage.py (hunk 1 above)
Comment (Gaspela) [reply models/engine/db_storage.py]: Ok, thank

**Review by Gaspela — COMMENTED** [review]
Ready changes

**Review by Gaspela — COMMENTED** [review]
Ready

**Pull request merged**